packages/portal-proto/src/features/cohortBuilder/dictionary.tsx

```diff
@@ -1,4 +1,4 @@
-import { useMemo } from "react";
+import { useDeepCompareEffect } from "use-deep-compare";
```

Comment:
Nitpick: didn't notice any JSX in this file. Might as well name `dictionary.tsx`.